### ISSUE TITLE
Program appears to be not responding when it is actually running fine

### ISSUE THREAD

**Issue by MilesWT** [MilesWT/USB-Benchmark#7]
It would be nice if we could figure out why this happens.
![image](https://cloud.githubusercontent.com/assets/11743094/25112195/a815d520-23bd-11e7-9b15-10f45ad11ebe.png)

**Comment by bdavs** [comment]
Yeah I don't think this one is worth the time.  Its something to do with a thread not completing in a given time.
